save-orchestrator/src/main/kotlin/org/cqfn/save/orchestrator/service/DockerService.kt

```diff
@@ -183,10 +183,12 @@ class DockerService(private val configProperties: ConfigProperties) {
             imageName = imageName(execution.id!!),
             baseDir = resourcesPath,
             resourcesPath = executionDir,
+            // TODO: find ktlint this is a temporary workaround link to #277
             runCmd = """RUN apt-get update && env DEBIAN_FRONTEND="noninteractive" apt-get install -y libcurl4-openssl-dev tzdata && rm -rf /var/lib/apt/lists/*
                     |RUN ln -fs /usr/share/zoneinfo/UTC /etc/localtime
                     |RUN chmod +x $executionDir/$SAVE_AGENT_EXECUTABLE_NAME
                     |RUN chmod +x $executionDir/$SAVE_CLI_EXECUTABLE_NAME
+                    |RUN find $executionDir/diktat-rules/src/test/resources/test/smoke/ktlint -type f -exec chmod +x {} \;
```

Comment:
```suggestion
                    |RUN find . -name $executionDir/diktat-rules/src/test/resources/test/smoke/ktlint -type f -exec chmod +x {} \;
```
I tried your command, and it returns 1 if no files have been found. This will cause the entire docker build to fail

Reply:
fixed